### PR TITLE
fix: prompts and segments schema

### PR DESCRIPTION
**Pull request by leogermani**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes the schema for prompts and segments.

Recently `criteria_hash` was added to segments, but we didn't update the schema, and thus we broke the importer/exporter tool. In prompts `duplicate_of` was in the wrong place.

### How to test the changes in this Pull Request:

1. on `master` in a site with many promtps and segments recently creted, try exporting it via CLI and confirm you see some error in segments: `wp newspack-popups export`
2. Checkout this branch and confirm export works
3. Also confirm import works

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "composer/installers": "^2.0",
     "brainmaestro/composer-git-hooks": "^2.8",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-    "yoast/phpunit-polyfills": "^1.0",
+    "yoast/phpunit-polyfills": "^1.1",
```
Comment (dkoo) [inline composer.json:12]: Are this and the related changes to `composer.lock` intended changes?

**Review by dkoo — APPROVED** [review]
Works! Just one non-blocking question.

File: composer.json (hunk 1 above)
Comment (leogermani) [reply composer.json:12]: Yes, otherwise phpunit won't run

Comment (dkoo) [reply composer.json:12]: Thanks! Just wanted to make sure because I've sometimes pushed unintended composer updates by accident.

**Pull request merged**